### ISSUE TITLE
gpg key used to sign aptly releases has expired

### ISSUE THREAD

**Issue by sford** [smira/aptly#364]
The gpg key used to sign aptly releases appears to have expired:

```
$ apt-key list | grep -A2 2A19
pub   2048R/2A194991 2014-03-12 [expired: 2016-03-11]
uid                  Andrey Smirnov <me@smira.ru>
```

This causes problems when trying to `apt-get update`:

```
W: GPG error: http://repo.aptly.info squeeze Release: The following signatures were invalid: KEYEXPIRED 1457723390 KEYEXPIRED 1457723390 KEYEXPIRED 1457723390
```

**Comment by scarytom** [comment]
I'm getting this too, the key has expired

http://keys.gnupg.net/pks/lookup?search=0x2A194991&op=vindex

**Comment by smira** [comment]
sorry for the trouble, I'm going to update the key tomorrow morning

**Comment by smira** [comment]
New key uploaded: https://groups.google.com/d/msg/aptly-discuss/Ny2NIqDgjHU/ZI6aNBeEAwAJ

**Comment by madwort** [comment]
Key has expired again!

```
apt-key list | grep -A2 9C7DE460
pub   2048R/9C7DE460 2016-03-15 [expired: 2018-03-15]
uid                  Andrey Smirnov <me@smira.ru>
```

**Comment by smira** [comment]
Getting this addressed, sorry #717
